src/test/java/millom/sandbox/service/WeatherServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package millom.sandbox.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import millom.sandbox.mapper.NasaMapper;
+import millom.sandbox.mapper.NasaMapperTest;
+import millom.sandbox.pojos.Weather;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+import resources.NasaResourceTest;
+
+@RunWith(MockitoJUnitRunner.class)
+class WeatherServiceTest {
+  @Mock
+  private MyClientService myClientService;
+  @Mock
+  private NasaMapper nasaMapper;
+  @InjectMocks
+  private WeatherService nasaResource = new WeatherService(myClientService,nasaMapper);
+
+
+  //public static final Weather WEATHER_EMPTY_DATA =
+
+  @BeforeEach
+  private void setUp() throws Exception{
+    MockitoAnnotations.initMocks(this);
```

Comment:
Not needed no?

Reply:
I am removing @RunWith(MockitoJUnitRunner.class) for now. Because it is not working as expected. Having only MockitoAnnotations.initMocks(this);